SparkPost/2web2ui
# SA-711: Spam Trap Hits by Type are viewable in the UI

Refer to [SA-711](https://jira.int.messagesystems.com/browse/SA-711)

### What Changed

<img width="661" alt="Screen Shot 2019-08-01 at 5 01 46 PM" src="https://user-images.githubusercontent.com/1335605/62327243-156f3780-b47e-11e9-9908-0961b818f2b9.png">

- Converted bar chart to stacked bar chart
- Added a legend
- Only displayed rate or count in tooltip (instead of both)
- Added types to tooltip

<img width="866" alt="Screen Shot 2019-08-02 at 10 34 45 AM" src="https://user-images.githubusercontent.com/1335605/62377670-36cd3380-b511-11e9-8543-0c7a1dff0149.png">

- Converted preview bart chart to stacked bar chart

### How To Test

_Our staging account does not have spam trap data, so ping me and I'll give you steps to get an account with data._

- Visit /signals/spam-traps/sid/1
- Confirm stacked bar chart is rendering for both "Count" and "Ratio"
- Confirm clicking on a bar changes the recommendations
- Confirm tooltip is providing the correct numbers for both "Count" and "Ratio"
